src/modules/users/user.controller.ts

```diff
@@ -0,0 +1,16 @@
+import { Request, Response } from 'express';
+import { validation } from '../../utilites';
+import { RegistrationUserDto } from './dto/registration-user.dto';
+import { UserService } from './user.service';
+
+const UserController = {
+  registration(req: Request, res: Response) {
+    const dto = validation(RegistrationUserDto, req.body);
+
+    const result = UserService.registration(dto, dto.nick);
```

Comment:
```suggestion
    const result = UserService.registration(dto);
```
А какой смысл отдельно передавать ник? Ты и так внутрь передаёшь весь dto, зачем отдельно ещё dto.nick передавать?